contracts/sol6/wrappers/KyberRateHelper.sol

```diff
@@ -172,39 +253,225 @@ contract KyberRateHelper is IKyberRateHelper, WithdrawableNoModifiers, Utils5 {
         uint256 optionalSellAmount,
         uint256 networkFeeBps
     ) internal view returns (bytes32[] memory sellReserves, uint256[] memory sellRates) {
-        Amounts memory A;
-        bool[] memory isFeeAccountedFlags;
-        address reserve;
+        uint256 sellAmount = optionalSellAmount > 0 ? optionalSellAmount : 1000;
```

Comment:
Can remove this line. `optionalSellAmount` can be renamed to `sellAmount`. Can even consider combining both `getBuyInfo` and `getSellInfo`, since the logic is almost the same, only difference is which reserve list to fetch from.